packages/x-domain/package.json

```diff
@@ -39,6 +39,7 @@
     "@eth-optimism/plugins": "^0.0.16",
     "@eth-optimism/provider": "^0.0.1-alpha.14",
     "@eth-optimism/rollup-core": "^0.0.1-alpha.28",
+    "@eth-optimism/solc": "0.7.6-alpha.1",
```

Comment:
Why did you need to add this here?

Reply:
@annieke from Slack:
> also had to upgrad solc to get their example contracts to compile but now it's breaking our merge checks

Reply:
originally i thought this was the only package that compiled solidity so i was working from it and later noticed it was from the base integration test `package.json`. i'll remove this, should i also remove it from tx-ingestion @tynes ?